src/services/lesson-service.js

```diff
@@ -22,7 +22,7 @@ class LessonService {
 
   async getLessonsMetadata(roomId) {
     const lessons = await this.lessonStore.getLessonsMetadataByRoomId(roomId);
-    return lessons.sort(by(l => l.schedule?.startsOn));
+    return lessons.sort(by(l => l.schedule?.startsOn || ''));
```

Comment:
Just a small thing: as we're comparing dates (which are compared as numbers), maybe `0` would be the better default value here than empty string?

Reply:
Indeed, changed to `0`, thanks!